doc/man3/EC_GROUP_new.pod

```diff
@@ -69,30 +66,14 @@ see L<openssl_user_macros(7)>:
                             const BIGNUM *a, const BIGNUM *b, BN_CTX *ctx);
  int EC_GROUP_get_curve_GFp(const EC_GROUP *group, BIGNUM *p,
                             BIGNUM *a, BIGNUM *b, BN_CTX *ctx);
- int EC_GROUP_set_curve_GF2m(EC_GROUP *group, const BIGNUM *p,
```

Comment:
Still more GF2m in this file (also in EC_GFp_simple_method.pod and EC_POINT_new.pod that should be mopped up.